### PR TITLE
Create Vision Zero issue template

### PR DESCRIPTION
**Pull request by johnclary**

### CODE REVIEW

**Comment by chiaberry** [comment]
changing us from assignees to reviewers

**Comment by johnclary** [comment]
i have no idea how to test this 🙃

File: .github/ISSUE_TEMPLATE/-all-purpose--vision-zero.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "[All-purpose] Vision Zero"
+about: Create a new Vision Zero issue
+title: "[<Type>] <Title>"
+labels: 'Workgroup: Vision Zero, Product: Vision Zero Crash Data System, Service: Dev'
+assignees: amenity
```
Comment (chiaberry) [inline .github/ISSUE_TEMPLATE/-all-purpose--vision-zero.md:6]: assignee, Amenity?

Comment (johnclary) [reply .github/ISSUE_TEMPLATE/-all-purpose--vision-zero.md]: thank you—fixed!

File: .github/ISSUE_TEMPLATE/-all-purpose--vision-zero.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: "[All-purpose] Vision Zero"
+about: Create a new Vision Zero issue
+title: "[Type] Title"
+labels: 'Workgroup: Vision Zero, Product: Vision Zero Crash Data System, Service: Dev'
+assignees: johnclary
```
Comment (chiaberry) [inline .github/ISSUE_TEMPLATE/-all-purpose--vision-zero.md:6]: 🥇

**Review by chiaberry — APPROVED** [review]
merge and test?

**Pull request merged**